modules/snooker/src/test/ts/browser/TableLookupTest.ts

```diff
@@ -76,94 +76,54 @@ UnitTest.test('TableLookupTest', function () {
   '</tr>' +
   '</tbody>' +
   '</table>';
-  const testTableRoWClick = function () {
+
+  const testTableRoWClick = () => {
     const triggerSelector = 'table > tbody > tr:nth-child(1) > td:nth-child(3) > table > tbody > tr:nth-child(1)';
-    testerShouldNotFind(htmlA, triggerSelector, 'testTableRoWClick');
+    testerShouldNotFind(html, triggerSelector, 'testTableRoWClick');
   };
 
-  const testOuterTableCellClick = function () {
+  const testOuterTableCellClick = () => {
     const triggerSelector = 'table > tbody > tr:nth-child(1) > td:nth-child(3)';
     const resultSelector = triggerSelector;
-    testerFound(htmlA, triggerSelector, resultSelector, 'testOuterTableCellClick');
+    testerFound(html, triggerSelector, resultSelector, 'testOuterTableCellClick');
   };
 
-  const testInnerTableCaptionClick = function () {
+  const testInnerTableCaptionClick = () => {
     const triggerSelector = 'table > tbody > tr:nth-child(1) > td:nth-child(3) > table > caption';
-    testerShouldNotFind(htmlA, triggerSelector, 'testInnerTableCaptionClick');
+    testerShouldNotFind(html, triggerSelector, 'testInnerTableCaptionClick');
   };
 
-  const testOuterTableCaptionClick = function () {
+  const testOuterTableCaptionClick = () => {
     const triggerSelector = 'body > table > caption';
-    testerShouldNotFind(htmlA, triggerSelector, 'testOuterTableCaptionClick');
+    testerShouldNotFind(html, triggerSelector, 'testOuterTableCaptionClick');
   };
 
-  const htmlB =
```

Comment:
Removed this from the test as it was a duplicate of the other HTML, so it was redundant.